pkg/cli/mirror/catalog_images.go

```diff
@@ -161,9 +163,11 @@ func (o *MirrorOptions) rebuildCatalogs(ctx context.Context, dstDir string, file
 			if err != nil {
 				return refs, fmt.Errorf("error parsing image %q: %v", OPMImage, err)
 			}
-			// Update registry so the existing OPM image can be pulled.
-			// QUESTION(jpower): is assuming an image is present in a repo with the same name valid?
+
 			opmImage.Registry = mirrorRef.Ref.Registry
+			if len(o.UserNamespace) != 0 {
+				opmImage.Namespace = strings.Join([]string{o.UserNamespace, opmImage.Namespace}, "/")
```

Comment:
Same thing here

Reply:
Gotcha. I need to make that change in a few places. Working on that now.

Reply:
Gotcha. I need to make that change in a few places. Working on that now.